src/routers/mailingList/constant.ts

```diff
@@ -0,0 +1,20 @@
+export const EVENT_NAMES = [
```

Comment:
Not sure if we need this...
It's just used to create `DEFAULT_EVENT_RESPONSES` right?

Reply:
Yeah we actually don't. Initially I was thinking we might